chunky/src/res/se/llbit/chunky/ui/render/HelpTab.fxml

```diff
@@ -23,10 +23,10 @@
       <Label text="D strafe right"/>
       <Label text="R move up"/>
       <Label text="F move down"/>
-      <Label text="U toggle fullscreen mode"/>
-      <Label text="K move forward x100"/>
-      <Label text="J move backward x100"/>
+      <Label text="K move forward"/>
+      <Label text="J move backward"/>
```

Comment:
Maybe just drop J and K then while you're at it.

Reply:
Having controls that work but undocumented is a bit annoying.. I could remove mention of them and maintain documentation elsewhere. Otherwise removing the actual bindings would probably be the best option.

Reply:
I will look at removing the actual bindings tomorrow in a separate PR. Removing mention of the bindings is probably the first step.